README.md

```diff
@@ -29,191 +29,191 @@ Version 2.0.0 of this collection supports `boto3 >= 1.15.0` and `botocore >= 1.1
 ### Connection plugins
 Name | Description
 --- | ---
-[community.aws.aws_ssm](https://github.com/ansible-collections/community.aws/blob/stable-2/docs/community.aws.aws_ssm_connection.rst)|execute via AWS Systems Manager
```

Comment:
`collection_prep_add_docs` must be run against branch `stable-2`.

`collection_prep_add_docs -b stable-2 -p .`

Reply:
oh right! fixed.